docker-compose.yml

```diff
@@ -19,12 +19,17 @@ services:
       - backend
 
   db:
-    image: couchbase/server-sandbox:7.0.0-beta
+    image: couchbase/server-sandbox:7.0.0
     ports:
       - "8091-8095:8091-8095"
       - "9102:9102"
       - "11210:11210"
     expose:
       - "8091"
+      - "8092"
```

Comment:
Not sure we need these actually, was there a reason  or issue you saw that made you change them?

Reply:
I found them useful when debugging. Possibly more useful in mix-and-match.yml tbh, can review.